app/scripts/lib/setupSentry.js

```diff
@@ -103,7 +103,17 @@ export default function setupSentry({ release, getState }) {
     environment,
     integrations: [new Dedupe(), new ExtraErrorData()],
     release,
-    beforeSend: (report) => rewriteReport(report),
+    beforeSend: (report) => {
+      if (getState) {
```

Comment:
This will effectively disable Sentry in the background, because `getState` is never set in the background.

Reply:
This is still a good idea, but we would need to ensure `getState` is set properly.

Reply:
okay, I think have done that here https://github.com/MetaMask/metamask-extension/pull/15313/commits/1eba37b2fb59f6ad57c601d3117217e08eff1b1a